qtpy/tests/test_qtcore.py

```diff
@@ -1,18 +1,51 @@
 """Test QtCore."""
 
+from datetime import datetime
+import sys
+
 import pytest
 
 from qtpy import PYQT5, PYQT6, PYSIDE2, PYSIDE6, PYQT_VERSION, PYSIDE_VERSION, QtCore
+from qtpy.tests.utils import not_using_conda
 
 
 def test_qtmsghandler():
     """Test qtpy.QtMsgHandler"""
     assert QtCore.qInstallMessageHandler is not None
 
 
-def test_DateTime_toPython():
+def test_qdatetime_toPython():
     """Test QDateTime.toPython"""
+    q_date = QtCore.QDateTime.currentDateTime()
     assert QtCore.QDateTime.toPython is not None
+    py_date = q_date.toPython()
+    assert isinstance(py_date, datetime)
+
+
+@pytest.mark.skipif(sys.platform.startswith('linux') and not_using_conda(),
+                    reason="Fatal Python error: Aborted on Linux CI when not using conda")
```

Comment:
```suggestion
@pytest.mark.skipif(
    sys.platform.startswith('linux') and not_using_conda(),
    reason="Fatal Python error: Aborted on Linux CI when not using conda")
```

Reply:
:+1: This matches the skips on our other tests, and plus PEP 8 / 79 chars and hanging indent just looks nicer.